qiskit_nature/second_q/mappers/linear_mapper.py

```diff
@@ -22,16 +22,21 @@
 import numpy as np
 
 from qiskit.opflow import PauliSumOp
-from qiskit.quantum_info.operators import Pauli, SparsePauliOp
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.quantum_info.operators import Pauli
```

Comment:
I'm curious over the split here of one being imported from quantum_info now directly and the other from operators. The import will work either way for both and if one prefers the shorter then

> from qiskit.quantum_info import Pauli, SparsePauliOp

Same it true elsewhere where Operator can be imported likewise. Its no biggie since either works, but since it got changed....

Reply:
Yeh I also wondered what triggered this. It might have been some automatic editor change or it was an artifact of a macro used to iterate over all files which needed changing. In any case, I agree that this can be reverted although it really should not matter :sweat_smile: